docs/user/dashboard/dashboard.asciidoc

```diff
@@ -228,6 +228,8 @@ To view the data field summary information, navigate to the field, then click *i
 [role="screenshot"]
 image::images/lens_data_info.png[Data summary window]
 
+Sometimes documents have no data for the field inspected: *Lens* will show a "Other" entry to represent these documents in the distribution chart.
```

Comment:
```suggestion
Sometimes documents have no data for the field inspected:. *Lens* shows an *Other* entry to represent these documents in the distribution chart.
```

Reply:
Can you add a screenshot that shows this?

Reply:
I made the screenshot for it, but then dropped as I thought it was too much. Will add it back! 📸